gcc/cp/contracts.h

```diff
@@ -75,6 +76,27 @@ struct contract_role
   contract_semantic axiom_semantic;
 };
 
+/* P2900 contract clasification */
```

Comment:
We should have a comment here that these need to be kept in sync with libstdc++/src/std{/experimental}/contracts.

Reply:
done